### PR TITLE
[DNM] The big (IfAllElseFails) april fools PR

### PR DESCRIPTION
**Pull request by mrhugo13**
## About The Pull Request
Discord_2021-03-31_18-07-58
If #6446, #6442, and #6444 are all incompatible for eachother when testmerging, TM this pr instead as it does what all the other ones do but all in one. If it isn't then just close this PR or something.

## Why It's Good For The Game
what the PR's for #6446 #6442 and #6444 said or whatever, including all of their about the pull requests, there is nothing new added ontop of here.

## Changelog
:cl: Vondiech
add: utter pain and misery in a last-minute PR
/:cl:

### CODE REVIEW

**Comment by mrhugo13** [comment]
woops that map edit wasn't supposed to go there.

File: html/changelogs/AutoChangeLog-pr-6439.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: "TheOneAndMightyRed(redory), Alterist"
+delete-after: True
+changes: 
+  - bugfix: "Powernets, Disposalnets and Atmosnets on most maps."
+  - bugfix: "Some camera dirs."
```
Comment (BraveMole) [inline html/changelogs/AutoChangeLog-pr-6439.yml:5]: the fuck is that

File: html/changelog.html [hunk 2]
```diff
@@ -1286,33 +1286,6 @@ <h3 class="author">grinkgo updated:</h3>
...
-				<li class="bugfix">Fixes damage delays for most relevant damage events.</li>
-			</ul>
-			<h3 class="author">TiviPlus updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="refactor">Adds the managed spawner controller</li>
-			</ul>
-			<h3 class="author">grinkgo updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="bugfix">Sulaco's medbay has a brand new layout.</li>
-			</ul>
```
Comment (BraveMole) [inline html/changelog.html:1315]: the fuck is that

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: html/changelog.html (hunk 2 above)
Comment (mrhugo13) [reply html/changelog.html:1315]: more stuff that got caught in me hastily slapping this PR together, give me just a few moments.

File: code/datums/emergency_calls/pmc.dm [hunk 3]
```diff
@@ -52,3 +52,41 @@
...
+	H.fully_replace_character_name(M.name, H.real_name)
+
+	if(original)
+		qdel(original)
+
+	print_backstory(H)
+
+	var/datum/job/J = SSjob.GetJobType(/datum/job/pmc/greytider)
+	H.apply_assigned_role_to_spawn(J)
+	to_chat(H, "<p style='font-size:1.5em'><span class='notice'>You are a greytider! Shove and bonk, until it is horizontal! You are friendly to the TGMC forces.</span></p>")
```
[the remark this answers is not in the document]
Comment (mrhugo13) [reply code/datums/emergency_calls/pmc.dm:92]: neither did i but oh well.

**Comment by mrhugo13** [comment]
the fucking changelog thing is being really fucking hold on

**Comment by mrhugo13** [comment]
there we go, that's the silly changelogs gone finally.

**Pull request closed without merging**